pages/data.js

```diff
@@ -221,6 +221,9 @@ export default class Explore extends React.Component {
                       : 'btn btn--primary btn--chart-toggle'
                   }
                 >
+                  <span className="btn--chart-toggle--icon">
+                    <img src={require('../images/' + datasets[datasetName].icon)} alt={datasets[datasetName].name} />
```

Comment:
I just reintroduced the code that was removed, but I noticed that dynamic requiring like this is [discouraged by eslint](https://github.com/benmosher/eslint-plugin-import/blob/v2.16.0/docs/rules/no-dynamic-require.md). Do you know why we're doing this rather than using [the `/static` directory that Next.js gives us for serving static assets](https://nextjs.org/docs#static-file-serving-eg-images)?

Reply:
Oh! I just noticed that we're using [next-optimized-images](https://github.com/cyrilwanner/next-optimized-images), and we need to require images rather than include them in `/static` to get the benefit of that.

Seems like one solution could be to [use `require.context`](https://github.com/cyrilwanner/next-optimized-images/issues/16#issuecomment-405556547) if that's something we're interested in exploring.

Reply:
I ended up [explicitly requiring each icon individually in datasets.js](https://github.com/texas-justice-initiative/website-nextjs/pull/93/files#diff-7e98b319add247afe349c5ce706bbc8cR1-R3) to address this issue.